src/common/config/index.ts

```diff
@@ -0,0 +1,56 @@
+/*eslint-disable */
+const MAP_SERVER = (window as any)._env_.MAP_SERVER;
+const PUBLISH_POINT = (window as any)._env_.PUBLISH_POINT;
+const CHANNEL = (window as any)._env_.CHANNEL;
+const VERSION = (window as any)._env_.VERSION;
+const REQUEST = (window as any)._env_.REQUEST;
+const SERVICE_PROTOCOL = (window as any)._env_.SERVICE_PROTOCOL;
+const SERVICE_NAME = (window as any)._env_.SERVICE_NAME;
+const ACTIVE_LAYER = (window as any)._env_.ACTIVE_LAYER;
+
+const EXPORTER_CONFIG = {
```

Comment:
use confd

Reply:
git ignore file?

Reply:
Relevant **window.__env__** props are come from confd - it is a confd usage.
confd generates js file which downloaded and run on client.
This file included in index.html (same principals as in image exporter).

In .gitignore this file **public/env-config.js** - confd generated.